docs/source/custom/package.rst

```diff
@@ -93,7 +94,40 @@ keywords should mention ``cobald`` for findability.
         ...
     )
 
-There are currently no ``entry_points`` used by :py:mod:`cobald`.
+Configuration Plugins
+*********************
+
+Packages can declare callables as plugins for the YAML configuration format.
+Plugins are added as ``entry_points`` of the ``cobald.config.yaml_constructors`` group,
+with a name to be used in configurations.
+For example, a plugin class ``ExtensionClass`` defined in ``mypackage.mymodule``
+can be made available as ``MyExtension`` in this way:
+
+.. code:: python3
+
+    setup(
+        ...,
+        entry_points={
+            'cobald.config.yaml_constructors': [
+                'MyExtension = mypackage.mymodule:ExtensionClass',
+            ],
+        },
+        ...
+    )
+
+This allows using the extension with YAML tag syntax as ``!MyExtension``.
+Extension are treated as callables and, depending on how their node is defined,
```

Comment:
I don't get the point what exactly a *node* is.

Reply:
YAML calls its elements "node" - mapping nodes, sequence nodes and so on. Not sure if that is clear to people not familiar with YAML. (I did not know it either.)

Reply:
Task: Reformulate to avoid YAML jargon.